docs/tutorials/r.md

```diff
@@ -2,8 +2,79 @@
 
 This guide shows how to create recipes for R packages with scripts to execute them with rattler-build.
 
+## Generating R Package Recipes
+
+The easiest way to get started with R packages is to use the built-in recipe generator:
+
+```bash
+rattler-build generate-recipe cran package_name
+```
+
+For example, to generate a recipe for the "dplyr" package:
+
+```bash
+rattler-build generate-recipe cran dplyr
+```
+
+This will create a recipe.yaml file with all the necessary dependencies and configuration for building the R package.
+
+```yaml title="recipe.yaml"
+package:
+  name: r-specsverification
+  version: 0.1.0
+
+source:
+  url:
+    - https://cran.r-project.org/src/contrib/SpecsVerification_0.5-3.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/SpecsVerification/SpecsVerification_0.5-3.tar.gz
+  sha256: 630fd876b51cb5e22061fa64dbb447c09e88c14e81fb801001ae18e969a4e6ec
+
+build:
+  number: 0
+  script:
+    interpreter: r
+    content: |
+      # Note, to install the package via source, we need to set the SRC_DIR environment variable
+      install.packages(Sys.getenv("SRC_DIR"), repos=NULL, type="source")
+
+requirements:
+  build:
+    - ${{ compiler('c') }}                # [not win]
+    - ${{ compiler('cxx') }}              # [not win]
+    - mingwpy                             # [win]
+    - ucrt                                # [win]
```

Comment:
This syntax `# [win]` does not work anymore with v1 recipes.

Instead, you need to use

```
if: win
then:
  - m2-filesystem
  - ...
```